EditorInitializer fix

astrelsky opened the pull request:
I don't see it documented anywhere that `PropertyEditor`'s get keyed to a `ThreadGroup`. Under unknown conditions the Swing thread's `ThreadGroup` may not be the `GhidraThreadGroup`.

resolves #4758

dragonmacher commented:
It is frustrating that they do not document why and how they use `ThreadGroupContext`.   This makes sense as a fix for the options editors.    Perhaps I don't understand the original Script Manager issue.  Or, perhaps that was a different issue altogether?

astrelsky commented:
> It is frustrating that they do not document why and how they use `ThreadGroupContext`. This makes sense as a fix for the options editors. Perhaps I don't understand the original Script Manager issue. Or, perhaps that was a different issue altogether?

Honestly at this point I don't know. I will never know because I had deleted the logs when I removed all previous installation artifacts. I have not encountered that problem since I purged everything but I will start digging if it ever comes back up. I was considering retitling the issue to just "rendering issues" but never did.

dragonmacher commented:
> Honestly at this point I don't know. I will never know because I had deleted the logs when I removed all previous installation artifacts. I have not encountered that problem since I purged everything but I will start digging if it ever comes back up. I was considering retitling the issue to just "rendering issues" but never did.

Regarding the options rendering issues, were you able to verify this change fixed those for you?

astrelsky commented:
> > Honestly at this point I don't know. I will never know because I had deleted the logs when I removed all previous installation artifacts. I have not encountered that problem since I purged everything but I will start digging if it ever comes back up. I was considering retitling the issue to just "rendering issues" but never did.
> 
> Regarding the options rendering issues, were you able to verify this change fixed those for you?

Yes
